### ISSUE TITLE
No video when playback

### ISSUE THREAD

**Issue by glamorous** [RasPlex/RasPlex#244]
It's been a couple of times after a period of not using rasplex (couple of hours). That suddenly no images are shown in my sreensaver and when I try to play something trough airplay or queue or even a episode. It starts to play but only the audio, not the video.

My log from the last hour where it occurred a couple of times when trying: http://pastebin.com/nA6b1iVL

My settings:
```
Size of Raspberry Pi: 512)
Type of SD card: SDHC class 10
Internet connection type: Ethernet (But PMS on Wifi)
Version of RasPlex: RC3
How did you install rasplex? auto-update
PMS version: 0.9.9.7.429-f80a8d6
Type of plex server: Macbook Pro 2009 (Wifi) with media files on NAS (Ethernet)
Power supply type and power rating: 2A
```

**Comment by hagen** [comment]
I cannot play video, as described here, from iPhone to RasPlex 0.5.1.
Raspberry Pi Model B, 512MB, screen saver set to Dim, and rebooted as instructed. All I get is audio. Video continues to play on the iOS device. Could I please ask your suggestions? Might this be resolved in 0.6.0 Pre-release?

**Comment by BradMeier** [comment]
The original bug report was about playback in general, and was tracked down to a memory leak in the background art / screensaver.  Airplay being broken is a known issue.
